### ISSUE TITLE
Level of focus for ontologies

### ISSUE THREAD

**Issue by ricklupton** [IndEcol/OpenScience#6]
I think we could be more specific about what we're aiming to do when discussing ontologies. I've tried to explain what I mean on the [IE Ontology page on the wiki](https://github.com/IndEcol/OpenScience/wiki/IE-Ontology#levels-of-ontologies).

- do you agree with my classification or have I missed something?
- is this a useful way to think about it?
- if so, which level should we start at? the top level of organisations/sources seems the easiest place to get started and would tie into the "data inventory" project.

@konstantinstadler @nheeren @stefanpauliuk @rjmyers do I need to mention people or will you see it anyway?

**Comment by nheeren** [comment]
Oh - I kind of overlooked that page. Is our wiki already too fragmented :) ?

I think it is a great article. Can we link it to the [manifesto](https://github.com/IndEcol/OpenScience/wiki/IE-Open-Science-Manifesto-(work-in-progress)) page and integrate / summarize it there? IMO the manifesto should reflect our research agenda (that we possibly will propose to the ISIE).

> @konstantinstadler @nheeren @stefanpauliuk @rjmyers do I need to mention people or will you see it anyway?

I think everybody [watching](https://github.com/IndEcol/OpenScience/watchers) (see top right corner) should receive notifications automatically.

**Comment by rjmyers** [comment]
Yes I agree with Niko that we should try to define our manifesto first, by summarising each topic and their challenges, before splitting up too much into individual working groups.

**Comment by stefanpauliuk** [comment]
Think that we should do both: Work on the different strands, just exchaning ideas, trying out things, while at the same time also looking at the big picture, for which the manifesto is a great platform.
I added a few more aspects that an ontology should cover to the wiki.
